camel/utils/token_counting.py

```diff
@@ -300,6 +300,60 @@ def count_tokens_from_messages(self, messages: List[OpenAIMessage]) -> int:
         return self.client.count_tokens(prompt)
 
 
+class LiteLLMTokenCounter(BaseTokenCounter):
```

Comment:
since actual `model_type` is `str` rather than `ModelType`, it doesn't follow `BaseTokenCounter`
```suggestion
class LiteLLMTokenCounter:
```